src/fidesops/service/connectors/query_config.py

```diff
@@ -440,6 +455,14 @@ def generate_query(  # pylint: disable=R0914
         return None
 
 
+class MicrosoftWithoutTuplesSQLServerQueryConfig(
```

Comment:
Just a quick note - I did like the original MicrosoftSQLServerQueryConfig name here, same for BigQuery -

Reply:
ah, this was a mistake in refactoring the name!